### PR TITLE
Remove unnecessary code

### PR DESCRIPTION
**Pull request by mikluke**

### CODE REVIEW

File: x/community/types/types.go [hunk 1]
```diff
@@ -29,16 +29,12 @@ const (
...
-	Owner         sdk.AccAddress `json:"owner"`
-	Title         string         `json:"title"`
-	PreviewImage  string         `json:"previewImage"`
-	Category      Category       `json:"category"`
-	Text          string         `json:"text"`
-	LikesCount    uint32         `json:"likesCount"`
-	DislikesCount uint32         `json:"dislikesCount"`
-	PDV           sdk.Int        `json:"pdv"`
-	CreatedAt     uint64         `json:"createdAt"`
+	UUID         uuid.UUID      `json:"uuid"`
```
Comment (megaherz) [inline x/community/types/types.go:32]: Well, will post PDV be calculated at the Theseus level?

File: x/profile/types/types.go [hunk 2]
```diff
@@ -39,13 +39,12 @@ func IsValidAvatar(str string) bool {
 
 // Public profile data
 type Public struct {
-	FirstName    string `json:"firstName"`
-	LastName     string `json:"lastName"`
-	Avatar       string `json:"avatar"`
-	Bio          string `json:"bio"`
-	Gender       Gender `json:"gender"`
-	Birthday     string `json:"birthday"`
-	RegisteredAt int64  `json:"registeredAt"`
```
Comment (megaherz) [inline x/profile/types/types.go:48]: The Frontend uses this field as a starting point in the user's PDV graph. How will it handle with this change?

File: x/community/types/types.go (hunk 1 above)
Comment (mikluke) [reply x/community/types/types.go:32]: Yes

File: x/profile/types/types.go (hunk 2 above)
Comment (mikluke) [reply x/profile/types/types.go:48]: Theseus knows when, and for posts from Genesis they should use something else

**Pull request merged**